docs/geedocs/5.2.3/answer/7160003.html

```diff
@@ -42,12 +42,19 @@ <h5>
     <p id="overview_5.2.3">New Features</p>
   </h5>
     <p><strong>C++11 support</strong>. With release 5.2.3 Open GEE C++ code now supports and defaults to compiling using the <code>-std=gnu++11</code> compile flag.  This default can be overridden and the C++ code can be compiled using the <code>-std=gnu++98</code> flag instead.  However, Open GEE 5.2.3 will be the last release to support compiling C++ code using the <code>-std=gnu++98</code> flag.  The subsequent releases after Open GEE 5.2.3 will require the use of the <code>-std=gnu++11</code> flag when compiling C++ code.</p>
-    <p><strong>Remove TCP/IP connections to PostgreSQL</strong>. With release 5.2.3 Open GEE code will now connect to PostgreSQL exclusively using Unix domain sockets. This will make it easier to secure PostgreSQL for security sensitive installations.</p>
+    <p><strong>Remove TCP/IP connections to PostgreSQL</strong>. With release 5.2.3 Open GEE code will now, by default, connect to PostgreSQL exclusively using Unix domain sockets. This will make it easier to secure PostgreSQL for security sensitive installations. For those upgrading from a previous release the following files will need to be changed before disabling PostgreSQL tcp listener:
```

Comment:
Does the user have to do this or It is done in code :  disable PostgreSQL tcp listener 
If the user has to do do we need to specify  how to do that ?

Reply:
The user has to do this and we can assume if they are going to do it they know enough about PostgreSQL to do this.  We could create a page in our wiki to guide them on possible ways to be more secure with open GEE PostgreSQL instance (too much to do in release notes here) but that should be a new issue.